ingestion_tools/scripts/importers/collection_metadata.py

```diff
@@ -0,0 +1,8 @@
+from common.finders import DefaultImporterFactory
+from importers.base_importer import BaseFileImporter
+
+
+class CollectionMetadataImporter(BaseFileImporter):
+    type_key = "collection_metadata"
```

Comment:
This also needs a plural key.
```suggestion
    type_key = "collection_metadata"
    plural_key = "collection_metadata"
```